requirements/extra/s3.txt

```diff
@@ -0,0 +1 @@
+boto3==1.*
```

Comment:
You should add `moto==3.*` as it is a missing requirement when I tried to run the tests.

Reply:
I think I added it to `test-requirements.txt` - maybe I didn't commit it.

Reply:
Sorry. You are right `moto` is in `test-requirements.txt`